src/java-runtime/java/mono/android/MonoPackageManager.java

```diff
@@ -42,6 +45,11 @@ public static void LoadApplication (Context context, ApplicationInfo runtimePack
 				String dataDir      = getNativeLibraryPath (context);
 				ClassLoader loader  = context.getClassLoader ();
 				String runtimeDir = getNativeLibraryPath (runtimePackage);
+				int localDateTimeOffset = (Calendar.getInstance ().get (Calendar.ZONE_OFFSET) + Calendar.getInstance ().get (Calendar.DST_OFFSET)) / 1000;
```

Comment:
Don't initialize the variable here, put this call in the `else` block below line 52 instead